ui/src/assets/dnd/tmp-upload-file.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="63" height="64" viewBox="0 0 63 64">
```

Comment:
the file name is starting with temp. Did you confirm this image with Sergey?

Reply:
No, i use this image as placeholder